docs/src/main/asciidoc/deploying-to-openshift-native-howto.adoc

```diff
@@ -0,0 +1,118 @@
+
+You can deploy your native {project-name} applications to {openshift} compiled to native executables by using the Docker build strategy.
+
+You must create a native executable for your application that targets the Linux AMD64 operating system.
+If your host operating system is different from this, create a native Linux executable by using a container runtime, for example, Docker or Podman.
+
+Your Quarkus project includes pregenerated Dockerfiles with instructions.
+If you want to use a custom Dockerfile, add the file to the `src/main/docker` directory or any location inside the module.
+Additionally, set the path to your Dockerfile by using the `quarkus.openshift.native-dockerfile` property.
+
+== Prerequisites
+
+* You have a Linux AMD64 system or an Open Container Initiative (OCI) compatible container runtime, such as Podman or Docker.
+* You have a Quarkus Maven project that includes the `quarkus-openshift` extension.
+* You are working in the correct OpenShift project namespace.
+
+== Procedure
+
+. Set the Docker build strategy in your `application.properties` configuration file:
++
+[source, properties]
+----
+quarkus.openshift.build-strategy=docker
+----
+. Enable container-based native builds:
++
+[source,properties]
+----
+quarkus.native.container-build=true
+----
+. Optional: Set the following properties in the `application.properties` file based on your environment:
+** If you are using an untrusted certificate, enable certificate trust for the `KubernetesClient`:
++
+[source,properties]
+----
+quarkus.kubernetes-client.trust-certs=true
+----
+** To expose the service and create an {openshift} route, set the following property:
++
+[source,properties]
+----
+quarkus.openshift.route.expose=true
+----
+** To use a custom Dockerfile instead of the pregenerated Dockerfiles, set the path to your custom Dockerfile:
++
+[source,properties,subs="attributes+,+quotes"]
+----
+quarkus.openshift.native-dockerfile=<path_to_your_dockerfile>
+----
+For example, to specify a custom Dockerfile named `Dockerfile.custom-native`:
++
+[source,properties]
+----
+quarkus.openshift.native-dockerfile=src/main/docker/Dockerfile.custom-native
+----
+
+** Specify the container engine:
+***  To build a native executable with Podman:
++
+[source,properties]
+----
+quarkus.native.container-runtime=podman
+----
+*** To build a native executable with Docker:
++
+[source,properties]
+----
+quarkus.native.container-runtime=docker
+----
+
+. Finally, build the native executable, package, and deploy your application to {openshift}:
++
+[source,shell,subs="attributes+,+quotes"]
+----
+./mvnw clean package -Pnative -Dquarkus.openshift.deploy=true
+----
+
+== Verification
+
+. Verify that an image stream and a service resource are created, and that the application is deployed.
+Use the {openshift} web console or the following {openshift} command-line interface (CLI) commands:
++
+[source,shell,subs="attributes+,+quotes"]
+----
+oc get is <1>
```

Comment:
OC tool is not among prerequisities? You can use kubectl for certain OCP tasks I think as well, I think you should mention that OC tool is a prerequisity